site/oauth2-examples-azure.md

```diff
@@ -0,0 +1,237 @@
+# Use Azure Active Directory (Azure AD) as OAuth 2.0 server
+
+We are going to test 3 OAuth flows:
+1. Access management ui via a browser :ballot_box_with_check:
+2. Access management rest api :construction:
+3. Access AMQP protocol :construction:
+
```

Comment:
GLOBAL comment - avoid using the word "we" how about "Let's text the following 3 OAuth flows:"